aws/eks/README.md

```diff
@@ -91,6 +91,19 @@ cd helpers
 ./write_kubeconfig $clusername $region
 ```
 
+## Access Konveyor UI
+1. export KUBECONFIG=`pwd`/{REPLACE_MYNAME}.kubeconfig
+   * Look at your `my_override_vars.yml` and see the name you chose for ```myname```
+1. kubectl port-forward svc/tackle-ui 7080:8080 -n konveyor-tackle
```

Comment:
If we are using ALB for ingress, do we need to port forward? We could use the url provided by ingress to access the Konveyor ui. Am I missing something?

Reply:
Good catch, thank you!

I wanted a note of port-forward as a reference for folks, but need to add that with ALB it's not needed.